.mise.toml

```diff
@@ -65,6 +65,22 @@ docker run \
 pocketsizefund/{{arg(name="service_name")}}:latest \
 """
 
+[tasks."application:build"]
```

Comment:
Same nitpick as before - I'd be inclined to have it be `application:service:build` so that an individual service gets built/pushed where something like `application:build` might be for all services or something more comprehensive.